integrationTests/package.json

```diff
@@ -0,0 +1,133 @@
+{
+  "name": "commerce-sdk-integration-test",
+  "version": "1.4.1-beta.0",
+  "description": "Salesforce Commerce Cloud SDK Integration Test",
+  "homepage": "https://developer.commercecloud.com/s/",
+  "private": true,
+  "scripts": {
+    "depcheck": "depcheck",
+    "lint": "eslint . --ext .ts,.js --quiet --fix",
+    "pretest": "npm run lint && npm run depcheck",
+    "test": "nyc mocha \"tests/*.test.*\"",
```

Comment:
Would this try to run `.test.helper.ts` files? You may want to change this to specifically `.test.ts` and `.test.js`

Reply:
I didn't know that .test.helper.ts was a thing.